webapp/src/components/header/settings-dialog/SettingsDialog.tsx

```diff
@@ -93,6 +93,19 @@ export const SettingsDialog: React.FC<ISettingsDialogProps> = ({ open, closeDial
                                     })}
                                 </AccordionPanel>
                             </AccordionItem>
+                            <Divider />
+                            <AccordionItem value="about">
+                                <AccordionHeader expandIconPosition="end">
+                                    <h3>About</h3>
+                                </AccordionHeader>
+                                <AccordionPanel>
+                                    <Body1 color={tokens.colorNeutralForeground3}>
+                                        Frontend version: {process.env.REACT_APP_SK_VERSION}<br/>
+                                        {process.env.REACT_APP_SK_BUILD_INFO}<br/>
+                                        Backend version: {serviceOptions.values.version}
```

Comment:
What do these default to if built locally?

Reply:
Blank for the first two and whatever the backend gives us (1.0.0 if nothing was specified while building the backend) for the last line.

I could put something like "Local build" for the frontend version if no version is provided.

The second line just doesn't appear if nothing is given.

Reply:
Ok that's reasonable. Maybe defaulting to "Not provided" or something along those lines would make sense.

Reply:
Putting in "-" if nothing is specified